packages/quantic/force-app/main/default/lwc/quanticResultLink/__tests__/data/knowledgeArticleResult.json

```
@@ -0,0 +1,9 @@
+{
+  "result": {
+    "clickUri": "http://coveo.com/",
```

It doesn't matter but it's better x)
```suggestion
    "clickUri": "https://coveo.com/",
```

---

do you prefer `http`? x)

---

thought I changed it my bad